app/components/jobseekers/account_survey_link_component/account_survey_link_component.scss

```diff
@@ -1,22 +1,30 @@
 @import 'base_component';
 
 .account-survey-cta {
+  text-align: center;
   bottom: 0;
-  box-sizing: border-box;
-  display: block;
-  position: sticky;
 
-  @include govuk-media-query($until: desktop) {
-    margin: 0 -15px !important;
-  }
+  .wrapper {
+    display: inline-block;
+    position: sticky;
+
+    @include govuk-media-query($until: desktop) {
+      margin: 0 -15px !important;
+    }
 
-  .account-survey-cta__content {
-    background-color: govuk-colour('orange');
-    color: govuk-colour('black');
-    padding: govuk-spacing(2);
+    .account-survey-cta__content {
+      background-color: govuk-colour('blue');
+      color: govuk-colour('white');
+      padding: govuk-spacing(3) govuk-spacing(4);
 
-    a {
-      color: govuk-colour('black');
+      a.govuk-button {
```

Comment:
should only the class selector here i.e remove the `a.`

Reply:
Done.